src/components/Container.js

```diff
@@ -368,7 +375,19 @@ const mapStateToProps = (state, {contextId, targetVerseText, sourceVerse}) => {
       alignedTokens: getVerseAlignedTargetTokens(state, chapter, verse),
       verseAlignments: getVerseAlignments(state, chapter, verse),
       verseIsValid: getIsVerseValid(state, chapter, verse,
-        normalizedSourceVerseText, normalizedTargetVerseText)
+        normalizedSourceVerseText, normalizedTargetVerseText),
+      groupMenu: {
+        toolsReducer: ownProps.tc.toolsReducer,
+        groupsDataReducer: ownProps.tc.groupsDataReducer,
+        groupsIndexReducer: ownProps.tc.groupsIndexReducer,
+        groupMenuReducer: ownProps.tc.groupMenuReducer,
+        translate: ownProps.translate,
+        actions: ownProps.tc.actions,
+        isVerseFinished: ownProps.toolApi.getIsVerseFinished,
+        contextId,
+        manifest: ownProps.projectDetailsReducer.manifest,
+        projectSaveLocation: ownProps.projectDetailsReducer.projectSaveLocation
```

Comment:
I realize we are still in the process of migrating code out of tC so I wouldn't call this a blocker. However, we need to stop directly accessing files in the tools. Instead we should proxy all reading (and writing) through tC.

For example:
```js
const {
      tc: {
        writeProjectData,
        readProjectData
      }
    } = this.props;

const writePromise = writeProjectData(relativePath, data);
const readPromise = readProjectData(relativePath);
```